clinica/pipelines/statistics_volume/statistics_volume_pipeline.py

```diff
@@ -0,0 +1,347 @@
+
+__author__ = "Arnaud Marcoux"
+__copyright__ = "Copyright 2016-2019 The Aramis Lab Team"
+__credits__ = ["Arnaud Marcoux"]
+__license__ = "See LICENSE.txt file"
+__version__ = "0.3.0"
+__maintainer__ = "Arnaud Marcoux"
+__email__ = "arnaud.marcoux@icm-institute.org"
+__status__ = "Development"
+
+
```

Comment:
Missing coding UTF-8 header (fixed in upcoming PR #63), __author__ & cie to remove

Reply:
Done